app/interfaces/api/entities/search_result.rb

```diff
@@ -37,81 +35,104 @@ class SearchResult < BaseEntity
 
       private
 
+      def id
```

Comment:
i'd be tempted to put in some method missing other forwarding/delegation here for  these repeated object calls:
e.g. 
```ruby
   def method_missing(method_name, *args, &block)
      if method_name.in?(%w(id uuid scheme scheme_type state court_name case_type total))
        object[method_name]
      else
        super
      end
    end

    def respond_to_missing?(method_name, include_private = false)
      object.respond_to?(method_name.to_sym, include_private) || super
    end
````